.npmignore

```diff
@@ -1,2 +1,3 @@
 *
-!lib/**
\ No newline at end of file
+!lib/**
```

Comment:
It doesn't look like there is a `lib` directory in this project.

Reply:
I guess that's true.